presto-main/src/test/java/io/prestosql/execution/buffer/BenchmarkBlockSerde.java

```diff
@@ -50,6 +49,8 @@
 import java.util.List;
 import java.util.Random;
 import java.util.concurrent.TimeUnit;
+import java.util.function.BiConsumer;
```

Comment:
These benchmarks were intentionally similar to `BenchmarkColumnReaders`. Could we keep them uniform? This most likely requires extraction of datagen to some external utility. However, we could then also benchmark Parquet in similar way and do head-to-head comparisons with different serde.

Reply:
Sure, we can do this as following PR